packages/magicbell/src/method.ts

```diff
@@ -136,7 +95,8 @@ function normalizeArgs({
     return urlData;
   }, {});
 
-  const url = getUrl(path, urlData);
+  // We don't encode atm because the backend doesn't support that in PUT /users/email:user@domain.com
+  const url = getUrl(path, urlData, { encode: false });
```

Comment:
We don't encode url entities, because the backend doesn't support that in `PUT /users/email:user@domain.com`